examples/Custom Block Page + CAPTCHA + first party/block.html

```diff
@@ -110,14 +110,18 @@ <h2 id="main-title">
     <script>
         (function setRefId() {
             var pxUuid = window._pxUuid;
+            var uuidVerifyRegExp = /^\{?[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}\}?$/i;
```

Comment:
why are curly braces allowed?
also, the following string will pass you validation: `{69b1925e-ffbe-11e8-87ea-4c32759f9dcd`, which is not great

Reply:
also, there's no point in doing this `[0-9a-fA-F]` if you use the `i` flag which makes everything case insensitive